### ISSUE TITLE
Get user profile from UAA?

### ISSUE THREAD

**Issue by ahrobinson** [PredixDev/predix-nodejs-starter#4]
How would I go about getting the user profile in the most best practice way? In other versions of predix seed, there was middleware setup that made a request to /check_token and it would give you back user info. Would I just setup middleware like before or is there another way to do this via passport?

Screenshot references: https://puu.sh/rwLXo/ec56c5c3dd.png
https://puu.sh/rwM06/71afe853b4.png

**Comment by gstroup** [comment]
Good question.  I don't think that is supported by passport.  You're talking about the UAA user_info endpoint right?  I think you would have to set up your own route in express to call that endpoint.  I'll look into passport support for user_info a little more.

**Comment by ahrobinson** [comment]
That is what I thought as well. Thanks for looking into it. I will look more into it as well and post if I find anything.

**Comment by manderson23** [comment]
This would be useful to have as an example since the predix-seed is now based on this code. 

The previous angular seed supported user_info while the new polymer seed does not.

**Comment by gstroup** [comment]
I agree, this would be helpful.  We'll work on adding a route to express, to demonstrate how this is done.
